pycrostates/cluster/_base.py

```diff
@@ -562,6 +571,9 @@ def predict(
     ):
         """Segment `~mne.io.Raw` or `~mne.Epochs` into microstate sequence.
 
+        Segment instance into microstate sequence using the segmentation smoothing
+        algorithm introduce in [1]_.
```

Comment:
Same here, looks like one extra leading space. Probably just the render on phone..